### PR TITLE
Support disable user

### PR DESCRIPTION
**Pull request by geokollias**

### CODE REVIEW

File: railib/rest.py [hunk 1]
```diff
@@ -63,15 +63,15 @@ def _contains_insensitive(items: list, term: str) -> bool:
 
 
 # Fill in any missing headers.
-def _default_headers(url: str, headers: dict = None) -> None:
+def _default_headers(url: str, headers: dict = None) -> dict:
```
Comment (bradlo) [inline railib/rest.py:66]: good catch, but i would fix this by changing contains_insensitive to operate on a dict.

Comment (geokollias) [reply railib/rest.py:66]: Right, I did that in the first commit https://github.com/RelationalAI/rai-sdk-python/pull/35/commits/61ece902b2812fcd4c5ece9a3f0ed28337f20d48. Then thought it'd make more sense for `_contains_insensitive` to accept a list. I'll change it back.

File: railib/rest.py [hunk 2]
```diff
@@ -80,7 +80,7 @@ def _default_user_agent() -> str:
     return f"rai-sdk-python/{__version__}"
 
 
-def _encode(data) -> str:
+def _encode(data) -> bytes:
```
Comment (bradlo) [inline railib/rest.py:83]: also good catch.

File: railib/rest.py (hunk 1 above)
Comment (bradlo) [reply railib/rest.py:66]: thanks, ill go ahead an approve assuming you will make that fix. this is a pretty bad bug on my part.

**Review by bradlo — APPROVED** [review]

File: railib/rest.py (hunk 1 above)
Comment (geokollias) [reply railib/rest.py:66]: done

**Pull request merged**